Motor motor1
Motor motor2
Motor motor3
Motor motor4

Servo pan
Servo tilt

sensor_msgs/Imu imu
sensor_msgs/NavSatFix gps
sensor_msgs/BatteryState battery

std_msgs/Header header
